pinot-common/src/main/java/org/apache/pinot/common/protocols/SegmentCompletionProtocol.java

```diff
@@ -123,6 +123,9 @@
   public static final String PARAM_MEMORY_USED_BYTES = "memoryUsedBytes";
   public static final String PARAM_SEGMENT_SIZE_BYTES = "segmentSizeBytes";
   public static final String PARAM_REASON = "reason";
+  // Controls whether servers upload segments to the controller during LLC segment completion protocol.
+  public static final String PARAM_SEGMENT_UPLOAD_TO_CONTROLLER = "enableSegmentUploadToController";
```

Comment:
Why do you need this? Configure your own split committer class and use that on the server. Enable split commits all the time